src/main/resources/schema/ans/0.10.5/collection.json

```diff
@@ -173,6 +173,9 @@
 
     "content_aliases": {
       "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.5/traits/trait_content_aliases.json"
+    },
+    "dynamic_items": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.5/traits/trait_adynamic_items.json"
```

Comment:
i think there's a typo in there, `trait_adynamic_items`

Reply:
nice catch. Thanks